documentation/assemblies/assembly-connecting-applications.adoc

```diff
@@ -19,8 +19,54 @@ To connect to the messaging service from outside the {KubePlatform} cluster, TLS
 
 The messaging protocols supported depends on the type of address space used. For more information about address space types, see  link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#con-address-space-messaging[Address space].
 
-//TODO: Add information about retrieving the CA certificate and using
-//that in the client examples.
+[id='client-getcer-{context}']
+== Retrieving the Self Signed CA Certificate
+
+If you opted for the `selfsigned` certificate provider type in your addressspace endpoint configuration, the generated CA
+that signed the adddressspace server certificate is required to when connecting messaging client application. The
```

Comment:
```suggestion
that signed the `AddressSpace` server certificate is required when connecting to the messaging client application. The
```

Reply:
Is the "+" symbol required in front of line 26?

Reply:
I don't see a `+`, are you referring to https://www.gnu.org/software/diffutils/manual/diffutils.html#Detailed-Unified